docs_user/modules/con_ceph-daemon-cardinality.adoc

```diff
@@ -7,15 +7,12 @@ colocated within the same node.
 ifeval::["{build}" != "upstream"]
 For more information, see link:https://access.redhat.com/articles/1548993[Red Hat Ceph Storage: Supported configurations].
 endif::[]
-The resulting topology depends on the available hardware, as well as the amount
-of {Ceph} services present in the Controller nodes that are going to be
-retired.
-As a general rule, the number of services that can be migrated depends on the
-number of available nodes in the cluster. The following diagrams cover the
-distribution of the {Ceph} daemons on the {Ceph} nodes where at least three
-nodes are required in a scenario that includes only RGW and RBD, without the
-{Ceph} Dashboard:
+The resulting topology depends on the available hardware, as well as the number
+of {Ceph} services in the Controller nodes that are going to be retired.
+The number of services that you can migrate depends on the number of available nodes in the cluster. The following diagrams show the distribution of {Ceph} daemons on {Ceph} nodes where at least 3 nodes are required.
 
+* The following scenario includes only RGW and RBD, without the {Ceph} dashboard:
```

Comment:
By any chance, are these scenarios covered in https://access.redhat.com/articles/1548993?

Reply:
That article is the only available resource atm, and the main problem there is that you have to infer such table and make sure you land in a supported scenario. The idea behind this section is to make it a bit easier for the OpenStack customers that have an already deployed overcloud (we deploy it, so we pretty much know what the main scenarios are).
We can plan some doc work to provide a KB, but definitely out of scope for FR1.